guides/common/modules/proc_integrating-a-generic-rfc-2136-compatible-remote-dns-server.adoc

```diff
@@ -0,0 +1,90 @@
+[id="integrating-a-generic-rfc-2136-compatible-remote-dns-server"]
+= Integrating a generic RFC 2136-compatible remote DNS server
+
+You can configure {ProductName} to integrate a remote DNS server that supports dynamic updates as defined in RFC 2136.
+In this case, {ProductName} uses the `nsupdate` utility to update DNS records on the remote server.
+
+
+.Prerequisites
+* The remote DNS service is configured.
+* The remote DNS service supports RFC 2136-compatible dynamic updates
+* You copied the `/etc/rndc.key` file from the external DNS server to `/etc/foreman-proxy/rndc.key` on the {ProjectServer} or {SmartProxyServer}.
```

Comment:
The `/etc/rndc.key` path feels too specific to me. This still feels a bit verbose, but what I'm thinking.
```suggestion
* The RNDC key file to connect to the external DNS server is placed in `/etc/foreman-proxy/rndc.key` on the {ProjectServer} or {SmartProxyServer}.
```

Reply:
I slightly rephrased it to avoid the word "external".